tests/test_flask.py

```diff
@@ -275,6 +277,16 @@ def test_new_project(self):
                 'backend': 'PyPI',
             }
 
+            falseData = {
+                'name': 'repo_manager',
+                'homepage': 'testurlrepo_manager',
+                'backend': 'PyPI',
+            }
+
+            output = c.post(
+                '/project/new', data=falseData, follow_redirects=True)
+            self.assertTrue('Please enter a URL.' in output.data)
```

Comment:
Maybe this should be moved up, we define data above ``falseData`` (which in python should be ``false_data``) but only use ``data`` below.